s3fs/core.py

```diff
@@ -1754,6 +1766,30 @@ class S3File(AbstractBufferedFile):
     retries = 5
     part_min = 5 * 2 ** 20
     part_max = 5 * 2 ** 30
+    # See: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/
+    # services/s3.html#S3.Client.put_object
+    additional_info_keys = [
+        "CacheControl",
```

Comment:
I am no expert here, I am not certain it's useful to reflect all of these parameters back at the server. I certainly would be quite surprised to see them show up in an `info` call, since they are not actually information about the file for the most part.